src/map/utils/synthutils.cpp

```diff
@@ -87,6 +87,13 @@ namespace synthutils
 
         if (ret != SQL_ERROR && _sql->NumRows() != 0 && _sql->NextRow() == SQL_SUCCESS)
         {
+            // Check content tag first
+            char* contentTag = (char*)_sql->GetData(19);
```

Comment:
Normally I'd push for not using `char*` to represent strings. `std::string` is what we'd use here - but I'm literally rewriting this in the next couple of days so it doesn't matter 👍